src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Main {
+    static final Random random = new Random();
+
+    public static void main(String[] args) {
+        Figure[] figures = new Figure[4];
```

Comment:
```suggestion
        Figure[] figures = new Figure[ARRAY_SIZE];
```

create constant ARRAY_SIZE = 6;

Reply:
no, it is the size of array of all possible figures and I have only four figures: Circle, Rectangle, Square and Isosceles Trapezoid